client/signup/steps/jpo-summary/index.jsx

```diff
@@ -0,0 +1,207 @@
+
+		const checkMark = <Gridicon
+			className="plan-features__item-checkmark"
+			size={ 18 }
+			icon="checkmark"
+			/>;
+
+		const {
+			signupProgress: {
+				0: {
+					jpoSiteTitle
+				},
+				1: {
+					jpoSiteType
+				},
+				2: {
+					jpoHomepage
+				},
+				3: {
+					jpoContactForm
+				}
+			}
+		} = this.props;
+
+		const pathPrefix = '/start/jetpack-onboarding/';
+
+		return (
+			<div className="jpo__summary-wrapper">
+				<table>
+					<tbody>
+						<tr>
+							<td>
+								<div className="jpo__summary-col-header jpo-summary__col-header">
+									{
+										translate( "Steps you've completed:" )
+									}
+								</div>
+								<ul className="jpo-summary__completed-onboarding">
+									{
+										jpoSiteTitle
+											? <li>{ checkMark } { translate( 'Site Title & Description' ) }</li>
+											: null
+									}
+									{
+										jpoSiteType
+											? <li>{ checkMark } { translate( 'Type of Site' ) }</li>
+											: null
+									}
+									{
+										jpoHomepage
+											? <li>{ checkMark } { translate( 'Type of Homepage' ) }</li>
+											: null
+									}
+									{
+										jpoContactForm
+											? <li>{ checkMark } { translate( 'Contact Us Form' ) }</li>
+											: null
+									}
+									{
+										connectionToJetpackComplete
+											? <li>{ checkMark } { translate( 'Connection to Jetpack' ) }</li>
+											: null
+									}
+								</ul>
+							</td>
+							<td>
+								<div className="jpo__summary-col-header">Configure more of your site:</div>
+								<ul className="jpo-summary__more-onboarding">
+									{
+										jpoSiteTitle
+											? null
+											: <li><a href={ `${ pathPrefix }jpo-site-title` }>{ translate( 'Site Title & Description' ) }</a></li>
+									}
+									{
+										jpoSiteType
+											? null
+											: <li><a href={ `${ pathPrefix }jpo-site-type` }>{ translate( 'Type of Site' ) }</a></li>
+									}
+									{
+										jpoHomepage
+											? null
+											: <li><a href={ `${ pathPrefix }jpo-homepage` }>{ translate( 'Type of Homepage' ) }</a></li>
+									}
+									{
+										jpoContactForm
+											? null
+											: <li><a href={ `${ pathPrefix }jpo-contact-form` }>{ translate( 'Contact Us Form' ) }</a></li>
+									}
+									{ ! connectionToJetpackComplete ? <li>{ translate( 'Connection to Jetpack' ) }</li> : null }
+									<li>{ translate( 'Choose a Theme' ) }</li>
+									<li>{ translate( 'Add a Site Address' ) }</li>
+									<li>{ translate( 'Add a Store' ) }</li>
+									<li>{ translate( 'Start a Blog' ) }</li>
+								</ul>
+							</td>
+						</tr>
+					</tbody>
+				</table>
+				<div>
+					<Button primary onClick={ this.completeOnboarding }>
+						{
+							translate( 'Complete Onboarding' )
+						}
+					</Button>
+				</div>
+			</div>
+		);
+	},
+
+	render() {
+		const headerText = translate( 'Congratulations! %s is on its way.', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **23** times:
`translate( 'Congratulations! %s is available.' )` *ES Score: 6*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).